puppet/modules/quickstack/manifests/pacemaker/neutron.pp

```diff
@@ -245,135 +245,162 @@
       resource_params => "clone globally-unique=true clone-max=${_clone_max} interleave=true",
       resource_type   => 'ocf',
     }
-    ->
```

Comment:
The diff is really bad here. Its just a if else statement, if "plumgrid" contains plumgrid related configuration and else contain the same old configuration.